cmd/set/puppet.go

```diff
@@ -0,0 +1,46 @@
+package set
+
+import (
+	"fmt"
+
+	"github.com/Masterminds/semver"
+	"github.com/puppetlabs/prm/pkg/prm"
+	"github.com/spf13/cobra"
+)
+
+type setPuppetCommand struct {
+	Config prm.ConfigI
+}
+
+var puppetSemVer *semver.Version
+
+func (spc *setPuppetCommand) createSetPuppetCommand() *cobra.Command {
+	tmp := &cobra.Command{
+		Use:     "puppet <VERSION>",
+		Short:   "Sets the Puppet runtime to the specified version",
+		Long:    `Sets the Puppet runtime to the specified version`,
+		PreRunE: spc.preExecute,
+		Run:     spc.setPuppetVersion,
+	}
+
+	return tmp
+}
+
+func (spc *setPuppetCommand) preExecute(cmd *cobra.Command, args []string) (err error) {
+	if len(args) != 1 {
+		return fmt.Errorf("incorrect number of args; please specify the puppet version as the only arg")
```

Comment:
Do we want to differentiate between "you did not pass an arg" and "you passed too many args" here?